### ISSUE TITLE
Switch to squash merge for cleaner commit history

### ISSUE THREAD

**Issue by chensun** [GoogleCloudPlatform/oss-test-infra#1437]
When I look at the change history under a folder, for example https://github.com/GoogleCloudPlatform/oss-test-infra/commits/master/prow/prowjobs/kubeflow/pipelines, it's not very easy to find a past change because the individual commits from a PR are not squashed. Can we switch to ["squash merge"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits) in the repo to have a cleaner change history?

**Comment by chaodaiG** [comment]
Yes this is supported, and I can see that kubeflow is already configured to do so https://github.com/GoogleCloudPlatform/oss-test-infra/blob/d99cc26484f4d88e77d9c37ca5a38887554fff6c/prow/oss/config.yaml#L31
